slack-api-client/src/main/java/com/slack/api/Slack.java

```diff
@@ -173,7 +173,7 @@ public String issueSocketModeUrl(String appToken) throws IOException {
         } catch (SlackApiException e) {
             String message = "Failed to connect to the Socket Mode API endpoint. (" +
                     "status: " + e.getResponse().code() + ", " +
-                    "error: " + e.getError().getError() +
+                    "error: " + (e.getError() != null ? e.getError().getError() : "") +
```

Comment:
`e.getError()` cannot be null in a usual situation... but, just in case, we want to improve this part not to throw NPE (I encountered with the mock proxy sever returning 502 status)